docs/source/intro.rst

```diff
@@ -0,0 +1,122 @@
+************
+Introduction
+************
+
+:Date: 2017-07-06
+:Authors: Chris Fonnesbeck
+:Contact: fonnesbeck@gmail.com
+:Web site: http://github.com/pymc-devs/pymc3
+
+
+Purpose
+=======
+
+PyMC3 is a probabilistic programming module for Python that allows users to fit Bayesian models using a variety of numerical methods, most notably, Markov chain Monte Carlo (MCMC) and variational inference (VI). Its flexibility and extensibility make it applicable to a large suite of problems. Along with core model specification and fitting functionality, PyMC3 includes methods for summarizing output, plotting, goodness-of-fit and model diagnostics.
+
+
+
+Features
+========
+
+PyMC3 strives to make Bayesian modeling as simple and painless as possible, thereby allowing users to focus on their scientific problem, rather than on the methods used to solve it. Here is a partial list of its features:
+
+* Fits Bayesian statistical models using modern techniques, including MCMC and VI.
+
+* Includes a large suite of well-documented statistical distributions.
+
+* Uses Theano as the computational backend, allowing for fast expression evaluation, automatic gradient calculation, and GPU computing.
+
+* Includes a module for Gaussian process modeling.
+
+* Creates summaries including tables and plots.
+
+* Several convergence diagnostics are available.
+
+* Extensible: easily incorporates custom step methods and unusual probability
+  distributions.
+
+* Bayesian models can be embedded in larger programs, and results can be analyzed
+  with the full power of Python.
+
+
+What's new in version 3
```

Comment:
Mabe just mention that it is a rewrite, but not much detail. That could be on a separate page pymc2 vs pymc3. I could be very wrong about this, but so far I didn't have the impression that most pymc3 users also knew pymc2. For those users the comparison is kind of pointless. Also, this kind of ends up repeating the feature list from above.
The important part of this is I think mentioned in the history section.